cognite/client/data_classes/transformations/common.py

```diff
@@ -23,7 +23,15 @@ def __eq__(self, other: Any) -> bool:
         return isinstance(other, type(self)) and hash(other) == hash(self)
 
     def dump(self, camel_case: bool = False) -> Dict[str, Any]:
-        return basic_obj_dump(self, camel_case)
+        ret = basic_obj_dump(self, camel_case)
+
+        for name, prop in ret.items():
+            if isinstance(
+                prop,
+                (ViewInfo, EdgeType),
+            ):
+                ret[name] = prop.dump(camel_case=camel_case)
```

Comment:
I'd prefer if you converted the specific properties instead of `isinstance`-checking everything. Something like:

```suggestion
        needs_dump = {"view", "edge_type"}
        for k in needs_dump.intersection(ret):
            ret[k] = ret[k].dump(camel_case=camel_case)
```

Reply:
I tried your suggestion but it breaks the test with this error:
```python
        if unknown_exc:
            if isinstance(unknown_exc, CogniteAPIError) and (failed or unknown):
>               raise CogniteAPIError(
                    message=unknown_exc.message,
                    code=unknown_exc.code,
                    x_request_id=unknown_exc.x_request_id,
                    missing=missing,
                    duplicated=duplicated,
                    successful=successful,
                    failed=failed,
                    unknown=unknown,
                    unwrap_fn=unwrap_fn,
                    extra=unknown_exc.extra,
                )
E               cognite.client.exceptions.CogniteAPIError: <exception str() failed>

```

Reply:
ok I figured it out should
```python
needs_dump = {"view", "edgeType"}
```
instead :)

Reply:
Oh, I totally forgot about camel case 🤦